tests/performance/collection_feed/weighted_set_feed.rb

```diff
@@ -139,11 +118,15 @@ def test_wset_attribute_feed_performance
     deploy_app(create_app)
     start
 
+    @data_generator = "#{dirs.tmpdir}/docs"
+    container = (vespa.qrserver["0"] or vespa.container.values.first)
+    container.execute("g++ -Wl,-rpath,#{Environment.instance.vespa_home}/lib64/ -g -O3 -o #{@data_generator} #{selfdir}/docs.cpp")
+
     doc_count = 10_000_000
     parameter_combinations.each do |p|
       # Reduce document count for large cardinalities to keep test time reasonable.
       test_doc_count = doc_count / p.wset_size
-      test_doc_count *= 5 if !p.fast_search
+      test_doc_count *= 2 if !p.fast_search
```

Comment:
Is it necessary to reduce the doc count now that we have a fast doc generator?

Reply:
Disregard, I just noticed the other PR that added the code in the first place... 🙂